contrib/masterfiles-stage/example_params/PARAMS_example_git_branch.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
-ROOT="/tmp/masterfiles_staging"
-GIT_URL="https://gitlab.com/nickanderson/masterfiles_demo_3.7.git"
-GIT_REFSPEC="master"
-# GIT_REFSPEC can be a git branch, tag, or commit hash.
-# It will be staged correctly regardless.
+ROOT="/opt/cfengine/masterfiles_staging"
+GIT_URL="https://gitlab.com/nickanderson/masterfiles-vcs"
+GIT_REFSPEC="3.15.x"
+GIT_USERNAME="nickanderson"
```

Comment:
do you want to include your username here?

Reply:
Ah, good I could remove that and it still works I think. Will check.

Reply:
yep.